tests/system/Config/fixtures/.env

```diff
@@ -6,4 +6,5 @@ NULL=
 
 SimpleConfig.onedeep=baz
 SimpleConfig.default.name=ci4
-simple.name=simpleton
\ No newline at end of file
+simple.name=simpleton
+impleconfig.shortprefix=wht
```

Comment:
Well that seems like an error then that should be fixed in BaseConfig :) Because no one's going to know to leave off the first letter. 

Man, it's been over a year since I've been in that code. :)